pydantic_ai_slim/pydantic_ai/durable_exec/temporal/_agent.py

```diff
@@ -57,6 +57,10 @@ def __init__(
         wrapped: AbstractAgent[AgentDepsT, OutputDataT],
         *,
         name: str | None = None,
+        additional_models: Mapping[str, Model | models.KnownModelName | str]
+        | Sequence[models.KnownModelName | str]
```

Comment:
Model strings that will just be passed to `infer_model` don't really need to be pre-registered, do they? So we can probably drop this type from the union